c7n/resources/dynamodb.py

```diff
@@ -229,15 +237,131 @@ def process(self, resources):
         if not len(resources):
             return
 
-        for table_set in chunks(resources, 20):
-            with self.executor_factory(max_workers=3) as w:
-                futures = []
+        futures = []
+
+        with self.executor_factory(max_workers=2) as w:
+            for table_set in chunks(resources, 20):
                 futures.append(w.submit(self.delete_table, table_set))
                 for f in as_completed(futures):
                     if f.exception():
                         self.log.error(
-                            "Exception deleting dynamodb table set \n %s" % (
-                                f.exception()))
+                            "Exception deleting dynamodb table set \n %s"
+                            % (f.exception()))
+
+
+@Table.action_registry.register('backup')
+class CreateBackup(BaseAction, StatusFilter):
+    """Creates a manual backup of a DynamoDB table. Use of the optional
+       prefix flag will attach a user specified prefix. Otherwise,
+       the backup prefix will default to 'Backup'.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: dynamodb-create-backup
+                resource: dynamodb-table
+                actions:
+                  - type: backup
+                    prefix: custom
+    """
+
+    valid_status = ('ACTIVE',)
+    schema = type_schema('backup',
+                prefix={'type': 'string'})
+    permissions = ('dynamodb:CreateBackup',)
+
+    def process(self, resources):
+        resources = self.filter_table_state(
+            resources, self.valid_status)
+        if not len(resources):
+            return
+
+        c = local_session(self.manager.session_factory).client('dynamodb')
+        futures = {}
+
+        if self.data.get('prefix'):
+            prefix = self.data.get('prefix')
```

Comment:
prefix = self.data.get('prefix', 'Backup')  is the simpler way of writing these lines.